src/link/Coff/lld.zig

```diff
@@ -98,6 +98,8 @@ pub fn linkWithLLD(self: *Coff, comp: *Compilation, prog_node: *std.Progress.Nod
         // strip does not need to go into the linker hash because it is part of the hash namespace
         man.hash.addOptional(self.base.options.major_subsystem_version);
         man.hash.addOptional(self.base.options.minor_subsystem_version);
+        man.hash.addOptional(self.base.options.version);
+        man.hash.addOptionalBytes(self.base.options.module_definition_file);
```

Comment:
This is a file path that the linker will read the contents of, yeah?

```suggestion
        man.hash.addOptionalFile(self.base.options.module_definition_file);
```

This will ensure that zig re-links if its contents changes.

Reply:
ah, whoops - fixed!